semantic-ast/src/Marshal/Examples.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE DataKinds, DeriveAnyClass, DeriveGeneric, DuplicateRecordFields, TypeOperators #-}
+module Marshal.Examples () where
```

Comment:
I don’t believe this is being compiled since it’s not included in the .cabal file.

Reply:
Yup!